### DISCUSSION TITLE
Saját adókódot kötelező-e megadni?

### DISCUSSION THREAD

**Discussion by Macskafarka** [nav-gov-hu/eVAT#83] [Kérdések és válaszok - Gépi interfész] [answered]
Az eÁFA_M2M_rendszer_interfesz_specifikacio_v0.5.pdf   leírás 25. oldalán az áll, hogy  a Standard adókon kívül meg kell adni "az adózó saját belső használatú adókódjait."  Nálunk ilyen belső használatú adókód nincs. A bevalláskor 21 db különböző mezőbeli értékből rakjuk össze. Ki kellene venni ezt a kötelezőséget, mivel adózói saját belső adókód használatra nincs jogszabályi előírás.

**Comment by renced42** [comment, marked as answer]
Kedves @Macskafarka 

A saját adókód mező most valóban kötelező, de ha nincs ilyen, akkor bármivel kitöltheted a séma szerinti valid értékkel.
Módosítjuk az XSD-t addig a fentiek szerint kell eljárni. Várhatóan az XSD módosítás január 31-g megtörténik, az éles rendszeren is.
